nutaudet/Target.cxx

```diff
@@ -388,28 +375,18 @@ void Target::ConstructGeometry()
   TGeoVolume *volBrick = new TGeoVolume("Brick",Brick,air);
   volBrick->SetLineColor(kCyan);
   volBrick->SetTransparency(1);
+  //need to separate the two cases, now with a ternary operator
+  auto *Absorber = new TGeoBBox("Absorber", EmulsionX/2, EmulsionY/2, LeadThickness/2);
+  auto *volAbsorber = (fDesign < 4) ?
+    new TGeoVolume("volAbsorber",Absorber,lead) : new TGeoVolume("volAbsorber",Absorber,tungsten);
```

Comment:
To avoid repeating the things that are the same:

```suggestion
  auto *volAbsorber = new TGeoVolume("volAbsorber", Absorber, (fDesign < 4) ? lead : tungsten);
```